Adding non-profit licensing program (#5541)
mattermost/docs: source/about/subscription.rst

Subscription overview
=====================

Mattermost offers features through `Mattermost Professional and Mattermost Enterprise subscription plans <https://www.mattermost.com/pricing>`__. Your plan subscription determines what features you have access to. When choosing a subscription, you can choose the deployment type as self-hosted or cloud. 

Mattermost self-hosted deployments require a license to be applied to access features in the Professional or Enterprise plans. Mattermost offers a `secure self-service Customer Portal <https://customers.mattermost.com>`__ where you can purchase and manage your Mattermost self-hosted subscriptions. When you purchase a subscription, a license is generated. Please see :doc:`Self-hosted subscriptions </about/self-hosted-subscriptions>` for more details. 

> Mattermost Cloud is a software-as-as-service. You can sign up and start using Mattermost in a trial mode, then upgrade to the edition you want to use within the product. Please see :doc:`Cloud subscriptions </about/cloud-subscriptions>` for more details.

Reviewer: ```suggestion
Mattermost Cloud is a software-as-a-service product. You can sign up and start using Mattermost in a trial mode, then upgrade to the edition you want to use within the product. Please see :doc:`Cloud subscriptions </about/cloud-subscriptions>` for more details.
```

Mattermost Non-Profit License Program

> ======================================

Reviewer: ```suggestion
----------------------------------------------------
```

> The Mattermost Nonprofit License enables eligible charities, open source projects and nonprofit organizations to apply the benefits of the self-hosted Mattermost Professional offering to advancing their missions with special nonprofit pricing.

Reviewer: ```suggestion
The Mattermost Nonprofit License enables eligible charities, open source projects, and nonprofit organizations to apply the benefits of the self-hosted Mattermost Professional offering towards advancing their missions with special nonprofit pricing.
```

> **What included?**

Reviewer: ```suggestion
**What's included?**
```

> A three-year subscription to our Mattermost Professional self-hosted offering for up to 1,000 users with a subscription fee of $250 USD. Upon acceptance of these terms, Mattermost has the right to place the name and logo of the nonprofit or charitable institution on our website and in our marketing materials.

Reviewer: @it33 - Is the $250 USD fee an annual subscription fee or a one-time cost?

Author: It's a fee of $250 in exchange for a 3-year subscription.

Following the three-year free license, the institution can renew the license every three years for an additional $250 USD subscription fee.

**Who's eligible?**

> To be eligible, an organization needs to be an official nonprofit or charity; as well as non-government, non-academic and non-commercial in nature, with no religious affiliation; and that would otherwise be unable to afford the commercial version of Mattermost software. If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>`__ instead.

Reviewer: ```suggestion
To be eligible, an organization needs to be an official nonprofit or charity; as well as non-government, non-academic, and non-commercial in nature, with no religious affiliation; and that would otherwise be unable to afford the commercial version of Mattermost software. If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>`__ instead.
```

> Mattermost will also have the right to place the name and logo of the nonprofit or charitable institution on our website and marketing materials. 

Reviewer: This line is duplicated above and no longer necessary.

Author: Updated, thanks @cwarnermm!

> **How can we apply?**

Reviewer: ```suggestion
**How to apply?**
```

To apply for the Mattermost Nonprofit License, `please complete this form <https://docs.google.com/forms/d/e/1FAIpQLSe2OUQGOQAni74FBXgAOooYsUTGbLn_VXFYEKUthg9Wmmiebg/viewform>`__.

> Mattermost Educational License Program

Reviewer: ```suggestion
Mattermost educational license program
```

Subscription overview
=====================

Mattermost offers features through `Mattermost Professional and Mattermost Enterprise subscription plans <https://www.mattermost.com/pricing>`__. Your plan subscription determines what features you have access to. When choosing a subscription, you can choose the deployment type as self-hosted or cloud. 

> Mattermost self-hosted deployments require a license to be applied to access features in the Professional or Enterprise plans. Mattermost offers a `secure self-service Customer Portal <https://customers.mattermost.com>`__ where you can purchase and manage your Mattermost self-hosted subscriptions. When you purchase a subscription, a license is generated. Please see :doc:`Self-hosted subscriptions </about/self-hosted-subscriptions>` for more details. 

Reviewer 2: ```suggestion
Mattermost self-hosted deployments require a license to be applied to access features in the Professional or Enterprise plans. Mattermost offers a `secure self-service Customer Portal <https://customers.mattermost.com>`__ where you can purchase and manage your Mattermost self-hosted licenses. When you complete the purchase process, a license is generated. Please see :doc:`Self-hosted subscriptions </about/self-hosted-subscriptions>` for more details.
```

Mattermost Cloud is a software-as-a-service product. You can sign up and start using Mattermost in a trial mode, then upgrade to the edition you want to use within the product. Please see :doc:`Cloud subscriptions </about/cloud-subscriptions>` for more details.

Mattermost Non-Profit License Program
------------------------------------------

The Mattermost Nonprofit License enables eligible charities, open source projects, and nonprofit organizations to apply the benefits of the self-hosted Mattermost Professional offering towards advancing their missions with special nonprofit pricing.

What's included?
~~~~~~~~~~~~

A three-year subscription to our Mattermost Professional self-hosted offering for up to 1,000 users with a subscription fee of $250 USD. Upon acceptance of these terms, Mattermost has the right to place the name and logo of the nonprofit or charitable institution on our website and in our marketing materials.

Following the three-year free license, the institution can renew the license every three years for an additional $250 USD subscription fee.

> **Who's eligible?**

Reviewer: ```suggestion
Who's eligible?
~~~~~~~~~~~
```

~~~~~~~~~~~~~~~~

To be eligible, an organization needs to be an official nonprofit or charity; as well as non-government, non-academic, and non-commercial in nature, with no religious affiliation; and that would otherwise be unable to afford the commercial version of Mattermost software. If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>`__ instead.

How to apply?
~~~~~~~~~~~~~~~

To apply for the Mattermost Nonprofit License, `please complete this form <https://docs.google.com/forms/d/e/1FAIpQLSe2OUQGOQAni74FBXgAOooYsUTGbLn_VXFYEKUthg9Wmmiebg/viewform>`__.

> 

Reviewer 2: ```suggestion
```

What's included?
~~~~~~~~~~~~

A three-year subscription to our Mattermost Professional self-hosted offering for up to 1,000 users with a subscription fee of $250 USD. Upon acceptance of these terms, Mattermost has the right to place the name and logo of the nonprofit or charitable institution on our website and in our marketing materials.

Following the three-year free license, the institution can renew the license every three years for an additional $250 USD subscription fee.

Who's eligible?
~~~~~~~~~~~

To be eligible, an organization needs to be an official nonprofit or charity; as well as non-government, non-academic, and non-commercial in nature, with no religious affiliation; and that would otherwise be unable to afford the commercial version of Mattermost software. If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>`__ instead.

Mattermost will also have the right to place the name and logo of the nonprofit or charitable institution on our website and marketing materials. 

> **How to apply?**

Reviewer: ```suggestion
How to apply?
~~~~~~~~~~
```

Mattermost self-hosted deployments require a license to be applied to access features in the Professional or Enterprise plans. Mattermost offers a `secure self-service Customer Portal <https://customers.mattermost.com>`__ where you can purchase and manage your Mattermost self-hosted subscriptions. When you purchase a subscription, a license is generated. Please see :doc:`Self-hosted subscriptions </about/self-hosted-subscriptions>` for more details. 

Mattermost Cloud is a software-as-a-service product. You can sign up and start using Mattermost in a trial mode, then upgrade to the edition you want to use within the product. Please see :doc:`Cloud subscriptions </about/cloud-subscriptions>` for more details.

Mattermost Non-Profit License Program
------------------------------------------

The Mattermost Nonprofit License enables eligible charities, open source projects, and nonprofit organizations to apply the benefits of the self-hosted Mattermost Professional offering towards advancing their missions with special nonprofit pricing.

What's included?

> ~~~~~~~~~~~~

Reviewer: ```suggestion
~~~~~~~~~~~~~~~~~
```

A three-year subscription to our Mattermost Professional self-hosted offering for up to 1,000 users with a subscription fee of $250 USD. Upon acceptance of these terms, Mattermost has the right to place the name and logo of the nonprofit or charitable institution on our website and in our marketing materials.

Following the three-year free license, the institution can renew the license every three years for an additional $250 USD subscription fee.

Who's eligible?

> ~~~~~~~~~~~

Reviewer: ```suggestion
~~~~~~~~~~~~~~~~
```

To be eligible, an organization needs to be an official nonprofit or charity; as well as non-government, non-academic, and non-commercial in nature, with no religious affiliation; and that would otherwise be unable to afford the commercial version of Mattermost software. If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>`__ instead.

Mattermost will also have the right to place the name and logo of the nonprofit or charitable institution on our website and marketing materials. 

How to apply?

> ~~~~~~~~~~

Reviewer: ```suggestion
~~~~~~~~~~~~~~~
```

To apply for the Mattermost Nonprofit License, `please complete this form <https://docs.google.com/forms/d/e/1FAIpQLSe2OUQGOQAni74FBXgAOooYsUTGbLn_VXFYEKUthg9Wmmiebg/viewform>`__.

Mattermost educational license program

> -------------------------------------

Reviewer: ```suggestion
----------------------------------------
```

> ======================================

Reviewer: ```suggestion
--------------------------------------------
```
